### DISCUSSION TITLE
WooPayments : Customize the boutons colors

### DISCUSSION THREAD

**Discussion by HawxFR** [woocommerce/woocommerce#39506] [General]
Hello,

I'm using the default payment plugin (WooPayments), but I can't customise the colours of the credit card or bank transfer buttons.
Look this : https://i.imgur.com/5v5Gf7T.jpg

Can you tell me how to do this?

Regards,
HawxNS

**Comment by HawxFR** [comment]
Hello,

Does anyone have any idea how to do this?

**Comment by thisissandip** [comment]
Hey @HawxFR! 👋

I understand that you would like to customize the payment fields and the buttons and apply some CSS on the form card form.

At this moment, the card field can’t be customized with CSS. This form is coming from `iFrame` directly from Stripe, and CSS will not apply here. Since the embed is coming from Stripe, this can’t be modified.

For assistance with such requests, I recommend submitting a support request on the dedicated plugin support forum. Specifically, for WooPayments, you can initiate a support request here: https://wordpress.org/support/plugin/woocommerce-payments/

Feel free to reach out on the forum to get the help you need. They'll be better equipped to provide guidance and solutions related to customizing parts coming from the plugin.
